.eslintrc

```diff
@@ -87,6 +87,15 @@
         "@shopify/jsx-no-hardcoded-content": "off",
         "@shopify/react-initialize-state": "off"
       }
+    },
+    {
+      "files": ["polaris.shopify.com/**/*"],
+      "extends": "plugin:@next/eslint-plugin-next/recommended",
```

Comment:
`eslint-config-next` enables the `plugin:@next/eslint-plugin-next/recommended` config and [a bunch of other things that are already covered by @shopify/eslint-plugin](https://unpkg.com/browse/eslint-config-next@12.1.4/index.js#L54). So here we just want to opt into the `plugin:@next/eslint-plugin-next/recommended` rules

Reply:
Will our `@shopify/eslint-plugin` rules take precedence? Or will they be overridden by `eslint-plugin-next/recommended`?